styles/layout.less

```diff
@@ -951,5 +951,47 @@ table.search-column {
     grid-row-gap: 2px;
 }
 
+.flex_container {
```

Comment:
Please add a section header like that on lines 944-945.

Reply:
Also removed unused parts which were pasted in from another branch